apps/server/src/modules/files-storage/service/files-storage.service.ts

```diff
@@ -227,7 +226,7 @@ export class FilesStorageService {
 			await this.deleteFilesInFilesStorageClient(fileRecords);
 		} catch (error) {
 			await this.fileRecordRepo.save(fileRecords);
-			throw new InternalServerErrorException(error, `${FilesStorageService.name}:delete`);
+			throw error;
```

Comment:
If we run in this cases, it is an InternalServerError.

Reply:
@CeEv This is correctly, we have all our errors

Reply:
ok